packages/gestalt/src/BannerOverlay/subcomponents.js

```diff
@@ -0,0 +1,148 @@
+// @flow strict
+import {
+  cloneElement,
+  type Element,
+  type ElementConfig,
+  Fragment,
+  type Node as ReactNode,
+  useCallback,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import Avatar from '../Avatar';
+import Box from '../Box';
+import Icon from '../Icon';
+import Image from '../Image';
+import Link from '../Link';
+import Mask from '../Mask';
+import Text from '../Text';
+
+const SIZE_THUMBNAIL = 40;
+const SIZE_ICON = 40;
+
+export function BannerOverlayMessage({
```

Comment:
can we reuse this component from Toast directly?

Reply:
DONE